man/Dockerfile.5.md

```diff
@@ -150,13 +156,20 @@ A Dockerfile is similar to a Makefile.
   the image.
 
 **LABEL**
-  -- `LABEL <key>[=<value>] [<key>[=<value>] ...]`
+  -- `LABEL <key>[=<value>] [<key>[=<value>] ...]`or 
+  ```
+  LABEL <key>[ <value>]
+  LABEL [<key>[ <value>]
```

Comment:
fixed, thanks

----- Original Message -----
From: "Patrick Hemmer" <notifications@github.com>
To: "docker/docker" <docker@noreply.github.com>
Cc: "sallyom" <somalley@redhat.com>
Sent: Thursday, September 3, 2015 8:33:46 AM
Subject: Re: [docker] man Dockerfile ADD/COPY/FROM clarify (#16043)

> @@ -150,13 +156,20 @@ A Dockerfile is similar to a Makefile.
>    the image.
>  
>  **LABEL**
> -  -- `LABEL <key>[=<value>] [<key>[=<value>] ...]`
> +  -- `LABEL <key>[=<value>] [<key>[=<value>] ...]`or 
> +  ```
> +  LABEL <key>[ <value>]
> +  LABEL [<key>[ <value>]

looks like an errant `[` before `<key>`

---
Reply to this email directly or view it on GitHub:
https://github.com/docker/docker/pull/16043/files#r38640170